images/image-alt-text-context-form.html

```diff
@@ -0,0 +1,164 @@
+
+                // Register a custom context toolbar shown when selecting any image
+                // https://www.tiny.cloud/docs/ui-components/contexttoolbar/#registeringacontexttoolbar
+                editor.ui.registry.addContextToolbar('imageContextToolbar', {
+                    predicate: function (node) {
+                        return node.nodeName.toLowerCase() === 'img'
+                    },
+                    items: 'alttext', // The context toolbar only contains our alt-text context form trigger button
+                    position: 'node',
+                    scope: 'node'
+                });
+
+                // The context form contains our alt text input field. It also manages
+                // the toolbar button that opens the context form.
+                // https://www.tiny.cloud/docs/ui-components/contextform/
+                editor.ui.registry.addContextForm('alttext', {
+                    // Define the button used in our context toolbar to open the context form
+                    launch: {
+                        type: 'contextformbutton',
+                        icon: 'alt-text',
+                        tooltip: 'Image description (Alt text)'
+                    },
+
+                    // The input field label (Currently does nothing).
+                    label: 'Image description (alt text)',
+
+                    // Initial value of the context form's input field
+                    initValue: () => {
+                        // First we get the selected element. It can be either the
+                        // callout itself or any child element within it.
+                        // https://www.tiny.cloud/docs/api/tinymce.dom/tinymce.dom.selection/#getnode
+                        const node = editor.selection.getNode();
+
+                        // Get the selected image's alt attribute text
+                        // https://www.tiny.cloud/docs/api/tinymce.dom/tinymce.dom.domutils/#getattrib
+                        const altText = tinymce.DOM.getAttrib(node, 'alt');
+
+                        // Return the the alt text to be displayed as initial value
+                        return altText;
+                    },
+                    // Define the buttons associated with the context form, in this case
+                    // a save and cancel button
+                    commands: [
+                        {
+                            type: 'contextformbutton',
+                            primary: true,
```

Comment:
Why doesn't this close the context form? I expected it to execute the `onAction` method but it seems not to do that. Or at least it ignores the hide action

Reply:
Caused by not naming parameters correctly